packages/seacas/libraries/ioss/src/text_mesh/Iotm_TextMeshUtils.h

```diff
@@ -704,14 +723,14 @@ class SidesetSplitter
   void set_split_type(SplitType inputSplitType) { m_splitType = inputSplitType; }
 
  private:
-  void create_index_proc_map(
+  void build_index_proc_map(
       const SidesetData<EntityId, Topology> &sideset, const std::vector<ElementData<EntityId, Topology>> &elementData)
   {
     for (size_t i = 0; i < sideset.data.size(); ++i) {
       const SidesetDataType<EntityId> &elemSidePair = sideset.data[i];
       EntityId elemId = elemSidePair.first;
 
-      auto iter = std::find(elementData.begin(), elementData.end(), elemId);
+      auto iter = bound_search(elementData.begin(), elementData.end(), elemId);
```

Comment:
OK.  That makes sense.  Is it ok that the sort uses a custom comparator, but the bound_search just uses the default less than comparator?

Reply:
That ..... is an interesting question. For the specific implementation of the custom comparator, it should be the same result as the default less because it is converting the ElementData entries into a simple EntityId. However, I see your point about consistency

Reply:
I'm adding an ElementDataLess comparator that can be used in the sort and search. Will push after I test it